File: In-memory-database/Level 3/C#/RecordField.cs
// RecordField.cs
public class RecordField
{
    public string Value { get; }
    public int? ExpiresAt { get; }

    public RecordField(string value, int? expiresAt = null)
    {
        Value = value;
        ExpiresAt = expiresAt;
    }

    public bool IsValid(int timestamp)
    {
        return ExpiresAt == null || timestamp < ExpiresAt;
    }
}

File: In-memory-database/Level 3/Python/record_field.py
# record_field.py

class RecordField:
    def __init__(self, value: str, expires_at: int | None = None):
        self.value = value
        self.expires_at = expires_at

    def is_valid(self, timestamp: int) -> bool:
        return self.expires_at is None or timestamp < self.expires_at
